llvm/lib/CodeGen/LiveVariables.cpp

```diff
@@ -246,6 +246,22 @@ LiveVariables::FindLastPartialDef(Register Reg,
   return LastDef;
 }
 
+static void fixupLiveIns(MachineInstr &MI, MCPhysReg SubReg) {
```

Comment:
This doesn't belong in this PR (and we should also delete LiveVariables)

Reply:
This code was necessary to get the added testcase to pass; otherwise it would crash in LiveVariables.